README.md

```diff
@@ -14,3 +14,18 @@ composer require bolt/configuration-notices
 ```
 
 ![7e22d8c8-bbdd-11e6-9668-48a1ec86ba04](https://cloud.githubusercontent.com/assets/1833361/21287029/3f4d8cf4-c463-11e6-8cd1-69583e7fa2ba.png)
+
+You can influence a few of the checks through the configuration. How meta! 
+
+```yaml
+configuration_notices:
+    local_domains: [ '.localhost' ]
+    lotsofrecords: 100
```

Comment:
`lotsofrecords` and the use of the phrase in function names is confusing, and semantically wrong.

What about `log_threshold: 100` … I immediately thought "to use "limit", but it is not a limit, it is just a threshold at which point you will warn the log is getting too big